megamek/src/megamek/common/MiscType.java

```diff
@@ -270,13 +270,14 @@
     public static final BigInteger F_CAPITAL_ARMOR = BigInteger.valueOf(1).shiftLeft(209);
     public static final BigInteger F_FERRO_CARBIDE = BigInteger.valueOf(1).shiftLeft(210);
     public static final BigInteger F_IMP_FERRO = BigInteger.valueOf(1).shiftLeft(211);
+    // Not usable by small support vehicles
+    public static final BigInteger F_HEAVY_EQUIPMENT = BigInteger.valueOf(1).shiftLeft(212);
```

Comment:
Question, is this going to affect saves (especially mekhq ones)?

Reply:
No. The only part of MiscType that gets serialized anywhere is te equipment lookup name.